test/test_calculate_level.py

```diff
@@ -0,0 +1,40 @@
+from serving.backend.calculate_level import calculate_level
+
+
+def test_calculate_level_standardized():
+    monster_adult_white_dragon = (
+        '{"cha":1, "con":5, "dex":2, "int":1, "str":7, "wis":2, "ac":29, "hp":215}'
+    )
+    monster_lantern_archon = (
+        '{"cha":1, "con":1, "dex":3, "int":-1, "str":-5, "wis":1, "ac":16, "hp":20}'
+    )
+    monster_treerazer = (
+        '{"cha":8, "con":11, "dex":9, "int":7, "str":12, "wis":8, "ac":54, "hp":550}'
+    )
```

Comment:
Zrobiłbym to jako `json.dumps(...)` i pisał dicty - będzie lepsza walidacja IDE. Jeżeli byłby jakikolwiek błąd w tym stringu JSONa, to PyCharm wam tego nie wykryje.